frontend/src/app/workspace/workspace.component.ts

```diff
@@ -29,18 +29,4 @@ export class WorkspaceComponent implements OnInit {
   ) {
     this.navbarService.title = 'Workspaces';
   }
-
-  ngOnInit() {
-    // FIXME: Fetch project users
-    // this.showSpinner = true;
-    // this.projectUserService.subscribe({
-    //   next: (res) => {
-    //     this.repositories = res;
-    //     this.showSpinner = false;
-    //   },
-    //   error: () => {
-    //     this.showSpinner = false;
-    //   },
-    // });
-  }
```

Comment:
Should this actually be removed because there is still a FIXME in it?

Reply:
In my opinion, it's a better way to track general issues in Github issues and use FIXME only for local markers, which are removed before merging. Otherwise there is the issue that FIXMEs and TODOs sum up and nobody cares about it anymore.